scalableinternetservices/ABCs
# Cacheresult

Test strategy for:
1) Load up the page and search for 'AAPL'
2) Search for the stock symbol again, it should load almost instantly
  a) it should also include one tweet's worth of data
3) Open rails console and type
 `cached = SentimentCache.find_by(company_id: 204)`
 `cached.created_at = #{current created_at minus one day}`
 `cached.save`
  a) This assumes AAPL's `company_id` is 204, which it should be
4) Then, search for the symbol again and it should take time to load everything from twitter

[EDIT] Unit tests now cover this
